monix-java/src/main/scala/monix/java8/eval/package.scala

```diff
@@ -30,7 +30,6 @@ package object eval {
   /**
     * DEPRECATED — switch to Scala 2.12+ and [[monix.eval.Task.from Task.from]].
     */
-  @deprecated("Switch to Scala 2.12+ and Task.from", "3.0.0")
```

Comment:
Why is this undeprecated?

Reply:
For the same reason. Scala 3 compiler triggers that warning even when not used. Don't know if this is a bug or what changed in Scala 3.

I have a zero tolerance for warnings :)